edx_exams/apps/api/v1/views.py

```diff
@@ -215,6 +215,19 @@ def handle_config(cls, provider, course_id):
         provider_name = provider.name if provider else None
         log.info(f"Created or updated course exam configuration course_id={course_id},provider={provider_name}")
 
+    def get(self, request, course_id):
```

Comment:
we should probably make a serializer for this if more fields get added but I chose not to go down that path since this was simpler and does't require altering the PATCH function.

Reply:
Do we expect they will stay in sync or get out of sync? If we want to make sure they don't get out of sync maybe a quick comment in each about "keep these in sync or make a serializer"

I can see patch taking more things than we are willing to provide to get. Although I guess that means they are really two separate endpoints rather than the same one.

Reply:
IMO these should stay in sync if they're at the same URL. I'll add a comment to note this for future us